$NetBSD$

IRIX 6.5 (mips64-sgi-irix): IRIX dlfcn.h values: RTLD_LAZY 1, RTLD_NOW 2, RTLD_GLOBAL 4, RTLD_LOCAL 0. Cargo.toml: only the dependencies IRIX builds use.

--- libloading-0.7.4/Cargo.toml.orig
+++ libloading-0.7.4/Cargo.toml
@@ -43,9 +43,3 @@ version = "1.1"
 [target."cfg(unix)".dependencies.cfg-if]
 version = "1"
 
-[target."cfg(windows)".dependencies.winapi]
-version = "0.3"
-features = [
-    "errhandlingapi",
-    "libloaderapi",
-]
